blazor/3D-chart/appearance.md

```diff
@@ -0,0 +1,559 @@
+        </Chart3DSeries>
+    </Chart3DSeriesCollection>
+</SfChart3D>
+
+@code{
+    public class Chart3DData
+    {
+        public string Country { get; set; }
+        public double Gold { get; set; }
+        public double Silver { get; set; }
+        public double Bronze { get; set; }
+    }
+
+    public List<Chart3DData> MedalDetails = new List<Chart3DData>
+	{
+         new Chart3DData{ Country= "USA", Gold=50, Silver=70, Bronze=45 },
+         new Chart3DData{ Country="China", Gold=40, Silver= 60, Bronze=55 },
+         new Chart3DData{ Country= "Japan", Gold=70, Silver= 60, Bronze=50 },
+         new Chart3DData{ Country= "Australia", Gold=60, Silver= 56, Bronze=40 },
+         new Chart3DData{ Country= "France", Gold=50, Silver= 45, Bronze=35 },
+         new Chart3DData{ Country= "Germany", Gold=40, Silver=30, Bronze=22 },
+         new Chart3DData{ Country= "Italy", Gold=40, Silver=35, Bronze=37 },
+         new Chart3DData{ Country= "Sweden", Gold=30, Silver=25, Bronze=27 }
+    };
+
+    public String[] palettes = new String[] { "#E94649", "#F6B53F", "#6FAAB0" };
+}
+
+```
+
+![Blazor Column 3D Chart with Custom Color Palette](images/appearance/blazor-column-chart-custom-color-palette.png)
+
+<!-- markdownlint-disable MD036 -->
+
+## Data point customization
+
+The color of an individual data point can be customized using the below options.
+
+### Point color mapping
+
+The color for the points can be bound from the `DataSource` for the series by utilizing the `PointColor` property.
+
+```cshtml
+
+@using Syncfusion.Blazor.Chart3D
+
+<SfChart3D Title="Olympic Medals" WallColor="transparent" EnableRotation="true" RotationAngle="7" TiltAngle="10" Depth="100">
+
+    <Chart3DPrimaryXAxis ValueType="Syncfusion.Blazor.Charts.ValueType.Category" />
+
+    <Chart3DSeriesCollection>
+        <Chart3DSeries DataSource="@MedalDetails" XName="X" YName="Y" PointColor="Color" Type="Chart3DSeriesType.Column">
+        </Chart3DSeries>
+    </Chart3DSeriesCollection>
+</SfChart3D>
+
+@code {
+    public class Chart3DData
+    {
+        public string X { get; set; }
+        public double Y { get; set; }
+        public string Color { get; set; }
+    }
+
+    public List<Chart3DData> MedalDetails = new List<Chart3DData>
+    {
+        new Chart3DData { X = "Jan", Y = 6.96, Color = "#ed4c40" },
+        new Chart3DData { X = "Feb", Y = 8.9,Color = "#3285f3"},
+        new Chart3DData { X = "Mar", Y = 12, Color = "#1dd7f3"},
+        new Chart3DData { X = "Apr", Y = 17.5, Color = "#fe1684" },
+        new Chart3DData { X = "May", Y = 22.1, Color = "#4633f2" }
+    };
+}
+
+```
+
+![Blazor Column 3D Chart with PointColorMapping](images/appearance/blazor-column-chart-with-point-color.png)
+
+## Point level customization
+
+The data label and fill color of each data point can be customized using the [`PointRender`](../api/chart3d/i3DPointRenderEventArgs/) and [`TextRender`](../api/chart3d/i3dtextrendereventargs/) events.
```

Comment:
@IlamathiSF3875 
In the description it says datalabel and fill customization but image and code example is not having that, point rendering only there, can you check and change?